client/components/Assignments/AssignmentEditor.tsx

```diff
@@ -17,6 +17,7 @@ import {
     SelectUserInput,
 } from '../UI/Form';
 import {ContactsPreviewList, SelectSearchContactsField} from '../Contacts';
+import {SelectUser} from 'superdesk-core/scripts/core/ui/components/SelectUser';
```

Comment:
Don't import things from superdesk-core. Instead consume it from superdesk api

`const {SelectUser} = superdeskApi.components;`

Reply:
I tried, but when we run testcases, we got  `SelectUser`  is `undefined`

Reply:
It will not work if you try to define that constant at the top level. Try definining it inside the component.

Reply:
![image](https://github.com/superdesk/superdesk-planning/assets/2076953/5d92b600-2672-487d-8c58-ca42dd27f5af)

Then you need to find where is it set, investigate why is it not set when running tests and resolve the issue.